src/services/api/contract_class.rs

```diff
@@ -43,6 +58,19 @@ impl From<&ContractEntryPoint> for Vec<MaybeRelocatable> {
     }
 }
 
+impl From<starknet_api::state::ContractClass> for ContractClass {
+    fn from(contract_class: starknet_api::state::ContractClass) -> Self {
+        let program = to_cairo_runner_program(&contract_class.program).unwrap();
+        let entry_points_by_type = convert_entry_points(contract_class.entry_points_by_type);
+
+        ContractClass {
+            program,
+            entry_points_by_type,
+            abi: None,
```

Comment:
Why the abi is set to None?

Reply:
It is not used during execution, we talked about it with @juanbono and decided to set it as None.